collections/_form/text-input.html

```diff
@@ -0,0 +1,186 @@
+        </li>
+
+    </ul>
+
+
+<hr>
+
+<h4 class="margin--top--lg margin--bottom has-text-weight-semibold">Error Messaging</h4>
+<div class="field">
+        <label class="label" for="email">Text Input Label</label>
+        <span class="error" for="email">This is a error message</span>
+        <div class="control">
+            <input
+                class="input is-danger"
+                type="email"
+                placeholder="Email input"
+                value="hello@example.com"
+                id="email"
+            />
+        </div>
+</div>
+
+
+
+<h5 class="has-text-weight-semibold margin--top--lg">Component Code</h5>
+{%- highlight html -%}
+<div class="field">
+        <label class="label" for="email">Text Input Label</label>
+        <span class="error" for="email">This is a error message</span>
+        <div class="control">
+            <input
+                class="input is-danger"
+                type="email"
+                placeholder="Email input"
+                value="hello@example.com"
+                id="email"
+            />
+        </div>
+</div>
+
+{%- endhighlight -%}
+<button
+    class="sgds-button clipboard-btn is-primary is-outlined"
+    data-clipboard-target=".highlight1">
+    Copy snippet
+</button>
+
+<h4 class="margin--top--lg has-text-weight-semibold">When you should use this component</h4>
+<p>Do use error messaging that is clear and concise. The error message should tell users what went wrong and/or how to recitify it.
+</p>
+
+<p>The error message should be inbetween the text input label and the text input field. If you have hint text, it should be inbetween
+    the hint text and the text input field.
+</p>
+
+
+<hr>
+
+<h4 class="margin--top--lg margin--bottom has-text-weight-semibold">Success Messaging</h4>
+<div class="field">
+        <label class="label" for="email">Email Address</label>
+        <span class="success" for="email">This is a valid email</span>
+        <div class="control">
+            <input
+                class="input is-success"
+                type="email"
+                placeholder="Email input"
+                value="hello@example.com"
+                id="email"
+            />
+        </div>
+</div>
+
+<h5 class="has-text-weight-semibold margin--top--lg">Component Code</h5>
+{%- highlight html -%}
+<div class="field">
+        <label class="label" for="email">Email Address</label>
+        <span class="success" for="email">This is a valid email</span>
+        <div class="control">
+            <input
+                class="input is-success"
+                type="email"
+                placeholder="Email input"
+                value="hello@example.com"
+                id="email"
+            />
+        </div>
+</div>
+
+{%- endhighlight -%}
+<button
+    class="sgds-button clipboard-btn is-primary is-outlined"
+    data-clipboard-target=".highlight1">
```

Comment:
@darenng this should be data-clipboard-target=".highlight**2**"

Reply:
Done, changed it to highlight2